_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -136,13 +138,15 @@
     .revenues_subpage-dyk_heading {
       background-color: $onshore-renewables-secondary-color;
     }
-    .revenues_subpage-steps_heading {
+    .revenues_subpage-steps_heading,
```

Comment:
Selector `revenues_subpage-steps_heading` should be written in lowercase with hyphens<br>Selector `revenues_subpage-steps_heading_more` should be written in lowercase with hyphens